### PR TITLE
Mysql optimize

### PR DESCRIPTION
**Pull request by edsonhor**
Added struct to facilitate ORM with mysql db.

### CODE REVIEW

File: pkg/kubernetes_clusters.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package clouddriver
```
Comment (billiford) [inline pkg/kubernetes_clusters.go:1]: Move this to package `kubernetes`, so we can reference it as `kubernetes.Cluster`.

File: pkg/kubernetes_clusters.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+type kubernetes_clusters struct {
```
Comment (billiford) [inline pkg/kubernetes_clusters.go:3]: In Go, we [export our structs](https://tour.golang.org/basics/3) that need to be visible outside their package. Additionally, all fields use Pascal case.

Rename this to `Cluster`.

File: pkg/kubernetes_clusters.go [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	account_name string
```
Comment (billiford) [inline pkg/kubernetes_clusters.go:4]: Make all of these fields Pascal case.

File: pkg/kubernetes_clusters.go [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+	cluster string
+	id string `json:"-" gorm:"primary_key"`
+	kind  string
+	name string
+	namespace string
+	spinnaker_app string
+
```
Comment (billiford) [inline pkg/kubernetes_clusters.go:11]: Remove this newline.

**Review by billiford — CHANGES_REQUESTED** [review]
See comments. Install a formatter that formats your code for you in [this fashion](https://github.com/homedepot/go-clouddriver/blob/master/pkg/kubernetes/resource.go).

File: pkg/kubernetes_clusters.go (hunk 1 above)
Comment (edsonhor) [reply pkg/kubernetes_clusters.go:1]: Sure

**Comment by edsonhor** [comment]
I m making the changes and resubmit

File: pkg/kubernetes/kubernetes_clusters.go [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package kubernetes
```
Comment (billiford) [inline pkg/kubernetes/kubernetes_clusters.go:1]: Rename this file from `kubernetes_clusters.go` to just `cluster.go`.

**Review by billiford — CHANGES_REQUESTED** [review]

**Pull request closed without merging**